spacy_llm/tasks/util/span_task.py

```diff
@@ -0,0 +1,102 @@
+from typing import Any, Callable, Iterable, List, Optional, Tuple
+
+import jinja2
+from spacy.tokens import Doc, Span
+
+from ...compat import Literal
+from ...registry import lowercase_normalizer
+from .examples import SpanExample
+from .parsing import find_substrings
+
+
+class AbstractSpanTask:
```

Comment:
`Base` seemed more appropriate to me?

```suggestion
class BaseSpanTask:
```

Reply:
Or just `SpanTask`?

Reply:
Thanks for the suggestion, it's committed.